features/bevragen/persoon/gezag/gba/gezag-minderjarige-gba.feature

```diff
@@ -0,0 +1,179 @@
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam | waarde             |
+      | type | GezagNietTeBepalen |
+
+
+  Rule: wanneer over een minderjarige tijdelijk niemand gezag heeft, wordt dat in gezag aangegeven
+
+    Scenario: tijdelijk heeft niemand gezag over de minderjarige
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                  | waarde   |
+      | geboortedatum (03.10) | 20190516 |
+      En de persoon heeft een ouder '1' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000024                   |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | burgerservicenummer (01.20) |
+      | 000000048                   |
+      En voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | G          |                  |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam | waarde             |
+      | type | TijdelijkGeenGezag |
+
+
+  Rule: wanneer een niet-ouder van rechtswege gezag heeft over de minderjarige wordt deze voogdij in gezag aangegeven 
+
+    Scenario: de partner van overleden ouder heeft gezag over de minderjarige
+      Gegeven voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | V          | 000000024        |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde    |
+      | type                             | Voogdij   |
+      | minderjarige.burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'derde' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+  Rule: wanneer van rechtswege gezag is toegewezen aan een voogd dan is de voogd niet bekend en wordt gezag geleverd zonder belanghebbenden
```

Comment:
is dit niet juist niet van rechtswege maar agv een gerechtelijke uitspraak?

Reply:
klopt, de rule was niet juist. Maak ervan "wanneer met een gerechtelijke uitspraak gezag is toegewezen aan een voogd dan is de voogd niet bekend en wordt gezag geleverd zonder de voogd(en)"